less/mixins.less

```diff
@@ -1 +1,38 @@
 // Mixins below variables
+.center() {
+    display: flex;
+    justify-content:space-evenly;
+    align-items: center;
+    }
+    .center2() {
+        display: flex;
+        justify-content:center;
+        align-items: center;
+        }
+  .border-radius(@radius) {
+    -webkit-border-radius: @radius;
+       -moz-border-radius: @radius;
+            border-radius: @radius;
+  }
+
+  .button-creator(@width, @height,@nav-font-color, white) {
```

Comment:
`white` shouldn't be in the parentheses here. You can simply set `color: white` in this mixin. Only parameters that will be passed to this mixin should be here. If it doesn't have an `@`, set it within the mixin itself.